### ISSUE TITLE
can't run build-command.

### ISSUE THREAD

**Issue by sassy** [dolanmiu/ng-cli-angular#2]
I can't run to build.

```
$ npm run build-electron

> angular-electron@1.0.0 build-electron /Users/sassy/dev/angular/angular-electron
> nge build

env: node\r: No such file or directory
```

cause error to run `nge build` .
Did I mistake how to use command?

**Comment by sassy** [comment]
environment:
* node: v6.9.5
* npm: v5.3.0
* OS: Mac OS X 10.12.6

**Comment by dolanmiu** [comment]
I get this error too on Mac OS X, on Windows it works fine, I am quite puzzled

Been trying to figure out why since yesterday

**Comment by dolanmiu** [comment]
It's most definately related to this:

https://github.com/nwjs-community/nw-builder/issues/75

**Comment by sassy** [comment]
I will investigate this problem.
This problem may be my environment.

thanks.

**Comment by dolanmiu** [comment]
I have a feeling that if i publish this `npm` package on my mac, it will make the line endings to `LF` (unix), which would fix this problem (according to that post above).

What you can try is running `npm run build` inside the `ng-cli-electron` folder to build it yourself, maybe that could work

I will try tomorrow on my mac

**Comment by sassy** [comment]
I see.
I will try it. Thanks.

**Comment by sassy** [comment]
I run `npm run build` inside the `ng-lic-electron` and run `../ng-cli-electron/dist/cli.js build`.
It worked!

thanks.

**Comment by dolanmiu** [comment]
Released version `1.1.1` which is built from my mac OS, which fixed it

Thanks!
